src/main/kotlin/no/digdir/organizationcatalog/mapping/Mappers.kt

```diff
@@ -72,6 +78,13 @@ fun OrganizationDB.updateWithEnhetsregisteretValues(org: EnhetsregisteretOrganiz
             else -> false
         }
```

Comment:
Tror organizationPrefLabel også bør inkluderes i sjekken om den skal oppdateres:
```suggestion
val prefLabelShouldBeUpdated =
  when {
    org.navn.isBlank() && organizationPrefLabel == null -> false
    prefLabel.isNullOrEmpty() -> true
    organizationPrefLabel?.value != null && prefLabel != organizationPrefLabel?.value -> true
    name != org.navn -> true
    else -> false
  }
```